cumulusci/tasks/salesforce/metadata_etl/permissions.py

```diff
@@ -0,0 +1,132 @@
+import xml.etree.ElementTree as XML_ET
+
+from cumulusci.core.exceptions import TaskOptionsError
+from cumulusci.tasks.salesforce.metadata_etl import (
+    MetadataSingleEntityTransformTask,
+    get_new_tag_index,
+)
+
+
+class AddPermissions(MetadataSingleEntityTransformTask):
+    entity = "PermissionSet"
```

Comment:
How hard would it be to make this usable for Profiles too?

Reply:
I think it is doable. Profiles and Permission Sets are largely the same at the metadata level; Profiles are just far more frustrating to pull via MDAPI.

This is the logic CaseMan require(d) for their installer and which they built. I'd like to extend it to cover Profiles, Permission Sets, and all of the components they permission.

Reply:
What I'm wondering is whether that could be done in the same task class or would require a different class for adding to profiles. If this one will remain for permission sets only then we might want a more specific name like AddPermissionsToPermissionSet

Reply:
As implemented, it will require at least a different facade class since it's one MDAPI entity per `MetadataSingleEntityTransformTask`. I'll change the name.